### PR TITLE
[android] 'native-modules-android' & 'native-components-android' docs consistency

### PR DESCRIPTION
**Pull request by PietroTambu**
Hi, just a minor fix.
There is a Java comment example in the Kotlin section. Maybe a copy/paste errata?

### CODE REVIEW

File: docs/native-modules-android.md [hunk 1]
```diff
@@ -294,7 +294,6 @@ Locate ReactNativeHost’s `getPackages()` method and add your package to the pa
 override fun getPackages(): List<ReactPackage> =
     PackageList(this).packages.apply {
         // Packages that cannot be autolinked yet can be added manually here, for example:
-        // packages.add(new MyReactNativePackage());
```
Comment (cortinico) [inline docs/native-modules-android.md:297]: Let's update this to:
```kotlin
// add(MyReactNativePackage())
```

**Review by cortinico — CHANGES_REQUESTED** [review]

File: docs/native-modules-android.md (hunk 1 above)
Comment (PietroTambu) [reply docs/native-modules-android.md:297]: Okay, what do you think about this?

In the Kotlin example:
```kotlin
override fun getPackages(): List<ReactPackage> =
    PackageList(this).packages.apply {
        // Packages that cannot be autolinked yet can be added manually here, for example:
        // add(MyAppPackage())
    }
```
In the Java example:

```java
@Override
  protected List<ReactPackage> getPackages() {
    @SuppressWarnings("UnnecessaryLocalVariable")
    List<ReactPackage> packages = new PackageList(this).getPackages();
    // Packages that cannot be autolinked yet can be added manually here, for example:
    // packages.add(new MyAppPackage());
    return packages;
  }
```

**Comment by PietroTambu** [comment]
@cortinico
Update both current and version 0.73

**Review by cortinico — APPROVED** [review]

**Comment by cortinico** [comment]
Thank you very much @PietroTambu

**Pull request merged**